website/docs/reference/integrations/datadog.md

```diff
@@ -28,19 +27,40 @@ You can choose to trigger updates for the following events:
 - feature-stale-off
 - feature-environment-enabled
 - feature-environment-disabled
+- feature-variants-updated
```

Comment:
Possibly out of scope for this PR but I think we should use `feature-environment-variants-updated` instead: https://github.com/Unleash/unleash/pull/4801#discussion_r1332094112

Reply:
How does it work today? **Does** this work? If we should use `feature-environment-variants-updated`, can we switch this line to that?

```suggestion
- feature-environment-variants-updated
```

Reply:
I don't think it does anymore. My latest `feature-variants-updated` events from my local instance are from 2022 and they predate variants per environment. I suggest simply making the change, addressed in https://github.com/Unleash/unleash/pull/4802/commits/dbfeef1e23556408e9922e95f377f93a488c4f82